app/gui/docs/CONTRIBUTING.md

```diff
@@ -102,6 +102,19 @@ setup:
   environment variable to explicitly enable regeneration of the interface files.
   The `flatc` is run as part of `build.rs` script of the `enso-protocol package.
 
+- **(Optional) The Latest version of `wasm-opt`**
+
+  The version of `wasm-opt` that comes bundled with `wasm-pack` can be out of
+  date and slow. We recommend to install a newer version from the
+  [binaryen releases](https://github.com/WebAssembly/binaryen/releases), or
+  [compile the newest version from scratch](https://github.com/WebAssembly/binaryen#building=).
+  `wasm-pack` will pick up the locally installed version automatically and use
+  it instead of the bundled one. Binaryen packages are also available through
+  some system package managers (apt/pacman/brew), check whether the version
+  there is newer than the bundled version of wasm-pack (which is true for all of
+  them as of wasm-pack 0.10.2, which bundles version 90 which was released Dec
+  2019).
```

Comment:
I am not entirely sure what to do? `wasm-opt` is taking long time on my Kubuntu. When the `./run` command prints
```
$ ./run view-profile interactive.json
[INFO]: Optimizing wasm binaries with `wasm-opt`...
```
I can find the `wasm-opt`location:
```
$ ps -Af | grep wasm-opt
~/.cache/.wasm-pack/wasm-opt-4d7a65327e9363b7/wasm-opt /tmp/enso-wasm/ide_bg.wasm -o /tmp/enso-wasm/ide_bg.wasm-opt.wasm -O
$ ls ~/.cache/.wasm-pack/
chromedriver-618468e6c4ba9730  wasm-bindgen-e63eccbd87203048  wasm-opt-4d7a65327e9363b7
```
Who installed this component into the cache? `rustup`? Probably as that is what our [contributing info suggests](https://github.com/enso-org/enso/blob/14a01c46359d23b5c95d706a6ecba887b6178c40/docs/CONTRIBUTING.md#getting-set-up-rust):
```
$ cargo +stable install wasm-pack --version 0.10.2
```
Shall I somehow use `cargo` to `install` newer version of `wasm-opt`?

Reply:
@JaroslavTulach 
`wasm-pack` automatically downloads `wasm-opt` if none is found in `PATH`. It downloads an old version that is particularly slow. `wasm-opt` is part of binaryen project (general WASM stuff, not specific to Rust), so there is no `cargo` command to install it. You can try downloading binary from [a release](https://github.com/WebAssembly/binaryen/releases) and exposing it to your PATH.

Reply:
@JaroslavTulach `apt install binaryen` should work on Kubuntu